index.Rmd

```diff
@@ -40,79 +40,72 @@ can_lang <- readr::read_csv("https://github.com/ttimbers/canlang/raw/master/inst
   readr::write_csv("data/can_lang.csv")
 ```
 
-# R, Jupyter, and the tidyverse
+# R and the tidyverse
+## Overview
 
-This is an open source textbook aimed at introducing undergraduate students to data science. It was originally written for the University of British Columbia's [DSCI 100 - Introduction to Data Science](https://ubc-dsci.github.io/dsci-100/) course. In this book, we define data science as the study and development of reproducible, auditable processes to obtain value (i.e., insight) from data. 
+This textbook aims to be an approachable introduction to the world of data science. In this book, we define *data science* as the study and development of reproducible, auditable processes to obtain value (i.e., insight) from data. At a high level, in this book, readers will learn how to 
```

Comment:
This is a really long line. We should use line breaks to limit the characters per line to 80 characters (RStudio has a nice vertical line that tells you where this is). In addition to limiting to 80 characters, I have just started learning about semantic line breaks (see https://sembr.org/) in which you add a line break after each substantial unit of thought. I think if we do this in our writing, our revisions will become easier to manage.